events_test/update_event_test.go

```diff
@@ -196,3 +198,23 @@ func TestUpdateEventRemoveCategories(t *testing.T) {
 	require.NoError(t, err)
 	require.NotContains(t, updatedEvent.Categories, category)
 }
+
+func TestUpdateEventIsInThePast(t *testing.T) {
+	t.Parallel()
+	company := test_util.CreateTestCompany(t)
+	chartKey := test_util.CreateTestChart(t, company.Admin.SecretKey)
+	client := seatsio.NewSeatsioClient(test_util.BaseUrl, company.Admin.SecretKey)
+	_, err := client.Seasons.CreateSeasonWithOptions(chartKey, &seasons.CreateSeasonParams{EventKeys: []string{"event1"}})
+	require.NoError(t, err)
+
+	err = client.Events.Update("event1", &events.UpdateEventParams{IsInThePast: shared.OptionalBool(true)})
+	require.NoError(t, err)
+	updatedEvent, err := client.Events.Retrieve("event1")
+	require.NoError(t, err)
+	require.True(t, updatedEvent.IsInThePast)
+
+	err = client.Events.Update("event1", &events.UpdateEventParams{EventParams: &events.EventParams{
```

Comment:
I wouldn't test events in the past being immutable - that's covered by the functional tests in `seatsio-core-v2`.